### DISCUSSION TITLE
TOC in Markor notes

### DISCUSSION THREAD

**Discussion by GFNIAE** [gsantner/markor#1780] [Q&A] [answered]
Hello,

I am getting used to markdown format and Markor app. Thanks for it ! Writing again gives me some new insights on my daily life.

About table of content, I have just discovered one could enable TOC with the option for "Markdown" format.

But such a table of content is useful only when you have a long note, and rather disturbing (and wasting space) in short notes.

Is there a way to display it only in note where we want it ? I read somewhere about [TOC] that makes TOC to appear... but even without [TOC] it is displayed.

**Comment by gsantner** [comment, marked as answer]
You can generally enable or disable Table of Contents, in Markor settings.  
Additionally, you can add [TOC]: # "Title" to your markdown file and you get the TOC rendered in that file wherever you want it. And so have a TOC only in the files you like them to be present.

**Comment by GFNIAE** [comment]
Hello gsantner,

Thanks for your quick reply. Actually I have read the "markor-markdown-reference.md" fil through and haven't found such a TOC example.

Could you give me more clues ? Thanks !

**Reply by gsantner** [reply]
I updated the answer, alright the reference doesn't have a example for this currently.

**Comment by GFNIAE** [comment]
PS. I have tried `<span id='toc' />` as it works in some other (extended) markdown editors, but here it seems that it is not interpreted as an auto table of content.

**Comment by GFNIAE** [comment]
Thanks, it works well ! my next goal is to be able to customize rendering in "read" mode, especially for the table of content (a little "heavy" with its frame).

**Reply by gsantner** [reply]
Feel free to! You can use the "Inject -> head" settings option for that, using [CSS](https://www.w3schools.com/css/).

**Comment by harshad1** [comment]
Is there a way of suppressing The TOC in a specific file?
